CMakeLists.txt

```diff
@@ -304,6 +304,10 @@ set(AVIF_CODEC_DEFINITIONS)
 set(AVIF_CODEC_INCLUDES)
 set(AVIF_CODEC_LIBRARIES)
 
+if(NOT libyuv_FOUND)
+    set(AVIF_SRCS ${AVIF_SRCS} src/scale/scale_impl.c src/scale/scale.c)
```

Comment:
A comment here or in src/scale/README.md would be nice. It should include the libyuv version/commit at which the sources were copied from, or at least where to look it up (in avif_scale.h right now).